### DISCUSSION TITLE
Sprache auf Deutsch umstellen?

### DISCUSSION THREAD

**Discussion by bugfixTK** [ciur/papermerge#470] [Q&A]
Hallo zusammen, ich bin jetzt schon ein paar Tage zugange mit der Installation. Soweit klappt es nun auch. Docker läuft bei mir inzwischen auf der Diskstation und auf meinem Mac. Nach ein paar Problemen, läuft Papermerge nun auch!! Aber eben auf Englisch. Jetzt wollte ich das ganze auf Deutsch umstellen. Nur ich bin zu blöde dazu! Leider finde ich auch keine Anleitung, welche so weit vorne anfängt, dass es klappt. Ich bekomme nur Fehler oder gar keine Rückmeldung aus dem Terminal am Mac. Mit dem Terminal auf der DS komme ich überhaupt nicht zurecht -.- 
Kann mir jemand helfen und hat vielleicht eine Anleitung die ganz am Anfang anfängt :) 
Schon mal vielen Dank an alle, die mich jetzt nicht auslachen!!!

**Comment by ciur** [comment]
Hallo bugfixTK,

Papermerge 2.1 is nur auf Englisch verfügbar.

**Comment by bugfixTK** [comment]
Vielen Dank für die schnelle Antwort! Da habe ich mich leider nicht richtig ausgedrückt -.- sitze da schon zu lange dran ...
Meinte die OCR Sprache. Also die, die Standard auf Englisch steht und man eigentlich andere Sprache auswählen kann.

**Comment by ciur** [comment]
ach so. Papermege 2.1 hast Deutsch und Englisch OCR schön drin (standardmäßig). Du kannst "Default OCR language" in "Preferences -> OCR" wählen:
![ocr-deutsch](https://user-images.githubusercontent.com/24827601/183256739-0c508792-0b47-4930-b9ca-dc57eaecb1e8.gif)

Antwortet dass deine Frage ?

**Comment by bugfixTK** [comment]
Ok, jetzt wird es spannend! Ich habe die 2.0 Version. Wie bzw wo bekomme ich die 2.1 Version geladen? 
Bis jetzt habe ich immer diese Version genutzt.
docker pull linuxserver/papermerge 

Wie vielmehr was muss ich nun eingeben?

Sorry, dass ich dumme Fragen stelle, aber es sind meine ersten Gehversuche in Docker. So ganz langsam verstehe ich das ja, was ich machen soll, aber vieles ist echt noch verwirrend für mich.

**Comment by ciur** [comment]
Here ist geschrieben wie bekommst du 2.1:
1. [mit docker-compose](https://docs.papermerge.io/Installation/docker-compose.html)
2. [mit portainer](https://docs.papermerge.io/Installation/portainer.html)
3. [nur mit docker](https://docs.papermerge.io/Installation/docker.html)

Bitte beachte dass 2.1 ist in alpha, dass heißt noch nicht "production ready".
